src/components/Footer/StyledFooter.js

```diff
@@ -9,7 +11,10 @@ const FooterContainer = styled(Container)``
 const ContactPanelRow = styled(Row)``
 
 // Individual social media contact column
-const ContactPanelColumn = styled(Col)``
+const ContactPanelColumn = styled(Col)`
+  justify-content: center;
+  padding: 30px;
```

Comment:
This padding is too large. Also, on RAD this is expressed as a percentage value.